src/toil/wdl/wdl_synthesis.py

```diff
@@ -300,8 +305,14 @@ def fetch_ignoredifs_chain(self, assignments, breaking_assignment):
         return False
 
     def write_main_jobwrappers_if(self, if_statement):
+        # check for empty if statement
+        if not if_statement:
+            return self.indent_docstring('        pass')
```

Comment:
This is not a docstring. Should the method just be called `indent` or something?